controllers/sandbox_controller_test.go

```diff
@@ -293,7 +293,7 @@ func TestReconcile(t *testing.T) {
 							},
 						},
 					},
-					ObjectMeta: sandboxv1alpha1.PodMetadata{
+					ObjectMeta: sandboxv1alpha1.TemplateObjectMetadata{
 						Labels: map[string]string{
 							"custom-label": "label-val",
 						},
```

Comment:
Please add a test for this new feature, e.g., create a Sandbox with a custom Service spec and verifies that the resulting Service object has the correct type and metadata. Also validate the case for updating Services.